templates/data/warehouse/index.html

```diff
@@ -0,0 +1,371 @@
+                advanced deployment and operations automation to help you get the most out of your PostgreSQL-based data warehouse."
+            }
+        },
+        {
+            "link": {
+                "content_html": "Learn more about Charmed PostgreSQL&nbsp;&rsaquo;",
+                "attrs": {
+                "href": "/data/postgresql"
+            }}
+        }
+    ]) 
+  }}
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Questions? Get answers</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container--3-2 is-cover">
+            {{ image(url="https://assets.ubuntu.com/v1/6e5dbcd1-questions-get-answers.jpg",
+                        alt="",
+                        width="1200",
+                        height="801",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <p>
+          Do you have a data warehouse project in mind and want to get advice on implementing PostgreSQL? Contact
+          us now to discuss your needs.
+        </p>
+        <div class="p-cta-block">
+          <a class="p-button--positive js-invoke-modal"
+             href="/contact-us"
+             aria-controls="data&ai-modal">Contact
+          us</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="grid-row--50-50">
+      <hr class="p-rule" />
+      <div class="grid-col">
+        <h2>Resources</h2>
+      </div>
+      <div class="grid-col">
+        <div class="grid-row">
+          <div class="grid-col-1">
+            <h3 class="p-heading--5">Whitepapers</h3>
+          </div>
+          <div class="grid-col-3">
+            <p>
+              <a href="https://ubuntu.com/engage/spark_online_data_hub">A guide to building an online datahub
+              (data lakehouse)</a>
+            </p>
+            <hr class="p-rule--muted" />
+            <p>
+              <a href="https://ubuntu.com/engage/choose-a-database">How to choose a database management
+              system</a>
+            </p>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="grid-row">
+          <div class="grid-col-1">
+            <h3 class="p-heading--5">Webinar</h3>
+          </div>
+          <div class="grid-col-3">
+            <p>
+              <a href="https://ubuntu.com/engage/data-lake-spark-openstack">A data lake on your cloud</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  {{ load_form("/solutions/data-and-ai") | safe }}
+
+  <style>
+    .p-table--mobile-card td.has-overflow {
+      text-wrap: auto;
+    }
+  </style>
```

Comment:
This isn't needed for the modal and can be removed. The `p-table--mobile-card` isn't a class that exists on the [modal template](https://github.com/canonical/canonical.com/blob/main/templates/shared/forms/form-template.html) so there's no relation there. Had a closer look and it looks like this snippet was copy pasted from /data/lakehouse where it's being used on one of the [tables](https://github.com/canonical/canonical.com/blob/721b75afce5e77dfa04d5b4ca7ab81af9f5f0404/templates/data/lakehouse/index.html#L214)
```suggestion
```